domain/model/state/state.go

```diff
@@ -307,6 +307,39 @@ SELECT type FROM model_type;
 	})
 }
 
+// List returns a list of all model UUIDs.
+// The list of models returned are the ones that are just present in the model
+// manager list. This means that the model is not deleted.
+func (s *State) List(ctx context.Context) ([]coremodel.UUID, error) {
+	db, err := s.DB()
+	if err != nil {
+		return nil, errors.Trace(err)
+	}
+
+	var models []coremodel.UUID
+	err = db.StdTxn(ctx, func(ctx context.Context, tx *sql.Tx) error {
```

Comment:
We'll need to remember to convert to sqlair. Since this is new an standalone, maybe we use sqlair here and add a todo for the other existing methods.

Reply:
I have to come back and do the whole state. Let me get all the DDL changes through. I haven't forgotten about this but I need to clear the backlog I have first.